WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/BarChartViewHolder.kt

```diff
@@ -42,6 +46,24 @@ class BarChartViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
         GlobalScope.launch(Dispatchers.Main) {
             delay(50)
             chart.draw(item, labelStart, labelEnd)
+            chart.post {
+                accessibilityHelper = BarChartAccessibilityHelper(
+                        chart,
+                        contentDescriptions = item.entryContentDescriptions,
+                        hasOverlappingEntries = item.overlappingEntries != null
+                )
+
+                accessibilityHelper.accessibilityEvent = object : BarChartAccessibilityEvent {
+                    override fun onHighlight(index: Int, hasOverlappingEntries: Boolean) {
+                        chart.highlightColumn(index, hasOverlappingEntries)
+                        val entry = chart.data.dataSets.first().getEntryForIndex(index)
+                        val value = entry?.data as? String
```

Comment:
Btw can't the caller pass us the value so we don't need to look it up and access the `chart.data.dataSet` from here?

Reply:
Yes, I will move this to the constructor.

Reply:
Resolved in 8bab45d dfbab1b

Reply:
Resolved df98ba8